festim/mesh/mesh_1d.py

```diff
@@ -61,3 +62,48 @@ def check_borders(self, volume_subdomains):
             or sorted_borders[-1] != self.vertices[-1]
         ):
             raise ValueError("borders dont match domain borders")
+
+    def define_meshtags(self, subdomains, volume_subdomains):
```

Comment:
I don't understand why we need both subdomains and volume_subdomains since one encompasses the other

Reply:
True